backend/__tests__/adminUpdateVendorRequests.test.js

```
@@ -0,0 +1,239 @@
+    expect(response.status).toBe(400);
+    expect(response.body.message).toBe('Invalid request ID');
+  });
+
+  it('should return 403 if the user is not an admin', async () => {
+    // FIXED: Temporarily override req.user to simulate non-admin
+    app.use('/api/admin/updateVendorRequest', (req, res, next) => {
+      req.user = { userId: 'testUserId', role: 'user' };
+      next();
+    }, (req, res, next) => {
+      // Reset after this test
+      delete req.user;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)